packages/frontend-2/components/header/NavLink.vue

```diff
@@ -1,14 +1,14 @@
 <template>
-  <div class="transition text-foreground hover:text-primary-focus">
+  <div class="text-foreground hover:text-primary-focus transition last:truncate">
     <NuxtLink
       :to="to"
-      class="flex items-center text-sm"
-      active-class="text-primary font-bold"
+      class="flex gap-1 items-center text-sm ml-0.5"
+      active-class="text-primary font-bold active-class"
```

Comment:
Weird class name, the `class` suffix is def redundant. Doesn't matter that much tho